webservice/bagitutils.py

```diff
@@ -322,3 +330,22 @@ def native_column_name(native_protocol, suffix):
     @staticmethod
     def firecloud_column_name(column):
         return column.lower().replace(' ', '_').replace('.', '_')
+
+    @staticmethod
+    def get_suffix(s):
+        """Lookup-table hack to address the problem of more than one file type 
+        occurring in a given file type column (2018-07-16).
+        :parameter s: (str) from a row from the manifest TSV file
+        :returns suffix: (str) holds integer corresponding to a file type"""
+
+        s = s.lower()
+
+        if s.endswith('fastq.gz') or s.endswith('fastq'): # unaligned read
```

Comment:
Did you verify these extensions with Francisco? I talked about this with Lon and he suggested there could also be fasta and sam file types in bioinformatics, but if U of C isn't handling them, then it's OK.